mining.md

```diff
@@ -265,10 +287,12 @@ To create a block, the eligible miner must compute a few fields:
 - `MsgRoot` - To compute this:
   - Select a set of messages from the mempool to include in the block.
   - Insert them into a Merkle Tree and take its root.
+    - Note: Messages with BLS signatures should be included as raw `Message` types, and not `SignedMessage`. Their signatures should be gathered up and aggregated for the `BLSAggregate` field. 
```

Comment:
What about ordering them and/or splitting them. Do you imagine these to be just interspersed as the miners choose?

Reply:
its up to the miner how they want to order messages. I could have a secp message that sends money to an account, and then next a BLS message that spends from that account. Or visa versa. Forcing one to come before the other doesnt work.